CMakeLists.txt

```diff
@@ -113,9 +97,33 @@ if(${CI_BUILD})
     endif()
 endif()
 
+# Link Time Optimization (LTO)
+cmake_policy(SET CMP0069 NEW)
```

Comment:
Do we need this? I thought using `cmake_minimum_required(VERSION 3.9 FATAL_ERROR)` automatically sets this policy.

Reply:
For me it works when leaving it out, but it probably doesn't hurt to explicitly set it anyway.